### ISSUE TITLE
New-line after closing brace/curly brace

### ISSUE THREAD

**Issue by Evangelink** [dotnet/roslyn#50195]
**Brief description:**

I would like to be able to enforce the presence of an empty line after a closing brace/curly brace (`if`, `else`, `for`, `foreach`...) and the next statement. Nothing should be raised if the next element is another closing brace or if it is part of a if/else if/else chain.

**Languages applicable:**: C#

**Code example that the analyzer should report:**

```csharp
void M()
{
	if (_myField.IsEnabled)
	{
		// Do something
	}
    Console.WriteLine("ok"); // report here or on previous line
}
```

Fixed code:
```csharp
void M()
{
	if (_myField.IsEnabled)
	{
		// Do something
	}

    Console.WriteLine("ok");
}
```

**Documentation requirements:**

When this analyzer is implemented, it must be documented by following the steps at [Documentation for IDE CodeStyle analyzers](https://github.com/dotnet/roslyn/blob/master/docs/contributing/Documentation%20for%20IDE%20CodeStyle%20analyzers.md).

**Comment by CyrusNajmabadi** [comment]
Closing out. Such an analyzer can already be written today. There is no need for it to be in roslyn.

**Comment by Evangelink** [comment]
I don't really understand your comment @CyrusNajmabadi. That's true for most of the code-style analyzers. Besides there are already plenty code-style rules related to placement, spacing and formatting of elements so I don't understand why this one wouldn't fit here.
